config.lua

```diff
@@ -8,7 +8,7 @@ config({'development'}, {
   }, 
   secret = "pleasechangeme",
   salt = "12",
-  body_size = "10m",
+  body_size = "20m",
```

Comment:
I'd apply the network engineering principle here, if we're approaching 50% then bump the limit. But your call :)